<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerAdd_UI</class>
 <widget class="QWidget" name="LayerAdd_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>199</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>376</width>
    <height>199</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>376</width>
    <height>199</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加图层</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/resource/img/logo.png</normaloff>:/img/resource/img/logo.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>图层属性</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>关联数据集</string>
    </property>
   </widget>
   <widget class="QComboBox" name="releativeDataSet">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="layerName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;新图层&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>图层名称</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="handIn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="layerType">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>影像</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数字高程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地表建筑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>巷道</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>钻孔</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>煤层</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>定位设备</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>监测设备</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工作面</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>断层</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>图层类型</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>handIn</sender>
   <signal>clicked()</signal>
   <receiver>LayerAdd_UI</receiver>
   <slot>slotHandIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>LayerAdd_UI</receiver>
   <slot>slotCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotHandIn()</slot>
  <slot>slotCancel()</slot>
 </slots>
</ui>
